[{"error_message": "cannot pickle 'tensorflow.python.client._pywrap_tf_session.TF_Operation' object", "function_to_run": "tf.keras.Sequential(*args)"}, {"error_message": "cannot pickle 'tensorflow.python.client._pywrap_tf_session.TF_Operation' object", "function_to_run": "obj.build(*args)"}, {"error_message": "Unknown layer: 'KerasLayer'. Please ensure you are using a `keras.utils.custom_object_scope` and that this object is included in the scope. See https://www.tensorflow.org/guide/keras/save_and_serialize#registering_the_custom_object for details.", "function_to_run": "obj.summary()"}, {"error_message": "invalid load key, '<'.", "function_to_run": "tf.io.read_file(*args)"}, {"error_message": "Unknown layer: 'KerasLayer'. Please ensure you are using a `keras.utils.custom_object_scope` and that this object is included in the scope. See https://www.tensorflow.org/guide/keras/save_and_serialize#registering_the_custom_object for details.", "function_to_run": "obj(*args)"}]